File: backend/src/main/java/com/smartcampus/backend/dto/freevehicle/FreeVehiclePositionDTO.java
package com.smartcampus.backend.dto.freevehicle;

import lombok.*;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
public class FreeVehiclePositionDTO {
    private String bikeId;
    private String vehicleTypeId;
    private Double latitude;
    private Double longitude;
}

File: frontend/app/src/main/java/com/ObservatoireCampus/mobile/model/freevehicle/FreeVehiclePositionDto.kt
package com.ObservatoireCampus.mobile.model.freevehicle

data class FreeVehiclePositionDto(
    val bikeId: String,
    val vehicleTypeId: String,
    val latitude: Double,
    val longitude: Double
)

data class VehicleTypeCountDto(
    val vehicleTypeId: String,
    val name: String,
    val count: Long
)